core/src/main/java/org/codice/imaging/nitf/core/common/impl/DateTimeParser.java

```diff
@@ -91,14 +91,13 @@ public final DateTime readNitfDateTime(final NitfReader reader) throws NitfForma
                 break;
             case UNKNOWN:
             default:
-                LOG.warn("Unknown NITF file type while reading date: " + reader.getFileType());
+                LOG.warn("Unknown NITF file type while reading date: {}", reader.getFileType());
                 throw new NitfFormatException("Need to set NITF file type prior to reading dates", reader.getCurrentOffset());
         }
         return dateTime;
     }
 
-    private void parseDateString(final String sourceString, final SimpleDateFormat dateFormat, final DateTimeImpl dateTime)
-            throws NitfFormatException {
+    private void parseDateString(final String sourceString, final SimpleDateFormat dateFormat, final DateTimeImpl dateTime) {
```

Comment:
This method mutates an input parameter. Generally not considered "good Java style". But I don't think it is worth refactoring at this point.